src/main/java/com/urswolfer/gerrit/client/rest/http/changes/ReviewInfoParser.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2013-2014 Urs Wolfer
```

Comment:
Please, change the year to 2012-2020

Reply:
No Problem, this should be done for the other files in a separate commit as well.

Reply:
Thank you for fixing it!
Yes, unfortunately in some files the copyright has not been updated for a while. It would be good to update all of them at one point.